File: CTDL1/New folder/Common7/IDE/ItemTemplatesCache/VisualBasic/Office SharePoint/1033/SharePoint16FullTrustVisualWebPart/Default.vb
Imports System
Imports System.ComponentModel
Imports System.Web
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports System.Web.UI.WebControls.WebParts
Imports Microsoft.SharePoint
Imports Microsoft.SharePoint.WebControls

<ToolboxItemAttribute(false)> _
Public Class $safeitemrootname$
    Inherits WebPart

    ' $loc_ascxPath_comment$
    Private Const _ascxPath As String = "~/_CONTROLTEMPLATES/15/$rootnamespace$/$fileinputname$/$fileinputname$UserControl.ascx"
    
    Protected Overrides Sub CreateChildControls()
        Dim control As Control = Page.LoadControl(_ascxPath)
        Controls.Add(control)
    End Sub

End Class

File: CTDL1/New folder/Common7/IDE/ItemTemplates/CSharp/Office SharePoint/1033/SharePoint15FullTrustVisualWebPart/Default.cs
using System;
using System.ComponentModel;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.Web.UI.WebControls.WebParts;
using Microsoft.SharePoint;
using Microsoft.SharePoint.WebControls;

namespace $rootnamespace$.$subnamespace$
{
    [ToolboxItemAttribute(false)]
    public class $safeitemrootname$ : WebPart
    {
        // $loc_ascxPath_comment$
        private const string _ascxPath = @"~/_CONTROLTEMPLATES/15/$rootnamespace$/$fileinputname$/$fileinputname$UserControl.ascx";

        protected override void CreateChildControls()
        {
            Control control = Page.LoadControl(_ascxPath);
            Controls.Add(control);
        }
    }
}
